src/test/util/mining.h

```diff
@@ -24,7 +24,7 @@ std::vector<std::shared_ptr<CBlock>> CreateBlockChain(size_t total_height, const
 CTxIn MineBlock(const node::NodeContext&, const CScript& coinbase_scriptPubKey);
 
 /** Prepare a block to be mined */
-std::shared_ptr<CBlock> PrepareBlock(const node::NodeContext&, const CScript& coinbase_scriptPubKey);
+std::shared_ptr<CBlock> PrepareBlock(const node::NodeContext&, const CScript& coinbase_scriptPubKey, bool check);
```

Comment:
https://github.com/bitcoin/bitcoin/pull/26695/commits/3f550b0887db0007633a26aac1dd80ec4723be55

Commit doesn't compile; looks like the `PrepareBlock()` call in `block_assemble.cpp` needs to be updated:
```cpp
bench/block_assemble.cpp:45:9: error: no matching function for call to 'PrepareBlock'
        PrepareBlock(test_setup->m_node, P2WSH_OP_TRUE);
        ^~~~~~~~~~~~
./test/util/mining.h:27:25: note: candidate function not viable: requires 3 arguments, but 2 were provided
std::shared_ptr<CBlock> PrepareBlock(const node::NodeContext&, const CScript& coinbase_scriptPubKey, bool check);
                        ^
1 error generated.
```